util.c

```diff
@@ -470,6 +470,12 @@ v4tov6(unsigned char *dst, const unsigned char *src)
     memcpy(dst + 12, src, 4);
 }
 
+int
+ae_is_v4(int ae)
+{
+    return ae == 1 || ae == 240;
```

Comment:
Why aren’t you using the `AE_` constants here? If you’re only using `ae_is_v4` in `message.c` you could put the function there, and you wouldn’t have to include `message.h` or to put the constant definitions in a weird place.

Reply:
I'm not using it anywhere else, but this looks like a function that might be needed somewhere else later on. I might be wrong here, though.

I did that a while ago, but I think that including `message.h` in `util` broke things and there was no trivial way to fix this, so it seemed easier to just do it this way.

Reply:
I don’t have a hard opinion on this, though it feels wrong not to use the `AE_` constants everywhere.

> that including message.h in util broke things

Does not surprise me :slightly_smiling_face: :

Reply:
This simply indicates that this function doesn't belong in util.  Util is for generic utility functions that are not closely tied to Babel.